src/components/Navbar/UserMenu.js

```diff
@@ -86,6 +86,37 @@ AdminMenuOptions.propTypes = {
     isMobile: PropTypes.bool.isRequired,
 };
 
+const CompanyMenuOptions = ({ isMobile, sessionData }) => {
+
+    const classes = useStyles({ isMobile });
+
+    if (sessionData.company.hasFinishedRegistration)
+        return (
+            <MenuItem button component={Link} to="/company/offers/manage" disableTouchRipple>My Offers</MenuItem>
+        );
+    else {
+        return (
+            <MenuItem
+                button
+                className={classes.highlightedMenuItem}
+                component={Link}
+                to="/company/registration/finish"
+                disableTouchRipple
+            >
+                <Badge
+                    variant="dot"
+                    color="primary"
+                    anchorOrigin={{ horizontal: "left", vertical: "top" }}
+                >
+                    <Box pl={2}>
+                        Finish Registration
+                    </Box>
+                </Badge>
+            </MenuItem>
+        );
+    }
```

Comment:
This is not very future proof (if we want to add a new option that shows regardless of that condition you would have to add it to both branches (if/else)

You can do it like this instead:
```
return (
<>
{sessionData.company.hasFinishedRegistration ? 
    <MyOffers stuff>
    : <FinishRegistration stuff>
}
// stuff that goes here shows every time
</>
)
```

Reply:
Also use `sessionData?.company?.hasFinishedRegistration` since if the logged in user is an admin, it would crash